### DISCUSSION TITLE
CSS doesn't work on my site but HTML does!

### DISCUSSION THREAD

**Discussion by Moatazodeh** [community/community#23718] [Other Feature Feedback, Questions, & Ideas]
<p>Hi everyone, its my first project in HTML and CSS and unfortunately its not working,<br>
I wrote it on Atom and when i open the website from a local link it works perfectly but when i open it with the Github link it looks like its without CSS, ive tried alot of things but still didn’t work…<br>
I will apperciate your help<br>
this is the link to my website : <a href="https://moatazodeh.github.io/website/" rel="noopener nofollow ugc">https://moatazodeh.github.io/website/</a><br>
and these are my files : <a href="https://github.com/Moatazodeh/website" rel="noopener nofollow ugc">https://github.com/Moatazodeh/website</a></p>

**Comment by airtower-luna** [comment, marked as answer]
<p>Your <code>index.html</code> includes the stylesheet like this:</p>
<pre><code class="lang-auto">    &lt;link rel="stylesheet" href="css/styles.css"&gt;
</code></pre>
<p>However in your repository the <code>CSS/</code> directory name is in capital letters. I assume you’re doing the local testing on Windows, because Windows generally ignores capitalization. Other platforms and browsers do not.</p>
<p>You need to either write the directory name in the <code>href</code> in capitals:</p>
<pre><code class="lang-auto">    &lt;link rel="stylesheet" href="CSS/styles.css"&gt;
</code></pre>
<p>Or alternatively rename the directory to be lower case.</p>

**Comment by Moatazodeh** [comment]
<p><a class="mention" href="/airtower-luna">@airtower-luna</a> thanks for answering it did work !</p>
